models/report.js

```diff
@@ -85,11 +88,23 @@ schema.methods.addSMTCTag = function(smtcTagId) {
   });
   if (isRepeat === false) {
     this.smtcTags.push({_id: smtcTagId});
+
+    if (!this.commentTo && this._media[0] === 'crowdtangle' && !this.url.match(/permalink/)) {
```

Comment:
This is a lot of conditions; looks fine but seems ripe for a nice "why" comment in the code.

Reply:
Comment added!